slope/index.js

```diff
@@ -130,6 +132,7 @@ parseData.fromCSV('./data.csv', dateStructure).then(({ seriesNames, groupNames,
         myChart.dotRadius(currentFrame.rem() * 0.3);
 
         // specifics based on frame type
+        myChart.setPalette(setColourPalette);
         myChart.colourPalette(frameName, groupNames); // set colour palette
```

Comment:
yup that's totally doable re passing in the argument. Changing colour to color should be a problem I'd just need to find and replace in all folders